example/main.go

```diff
@@ -11,31 +11,29 @@ import (
 )
 
 var (
-	portals   = flag.String("portals", "192.168.1.112:3260", "Comma delimited.  Eg: 1.1.1.1,2.2.2.2")
-	iqn       = flag.String("iqn", "iqn.2010-10.org.openstack:volume-95739000-1557-44f8-9f40-e9d29fe6ec47", "")
-	multipath = flag.Bool("multipath", false, "")
-	username  = flag.String("username", "3aX7EEf3CEgvESQG75qh", "")
-	password  = flag.String("password", "eJBDC7Bt7WE3XFDq", "")
-	lun       = flag.Int("lun", 1, "")
-	debug     = flag.Bool("debug", false, "enable logging")
+	portals  = flag.String("portals", "192.168.1.112:3260", "Comma delimited.  Eg: 1.1.1.1,2.2.2.2")
```

Comment:
At some point all these defaults should be removed and we should a mandatory check that the flags are set.  These dfaults are nice for an example but they won't mean anything to to someone that's not using the openstack test setup I had 3 years ago :)

Reply:
sure @j-griffith , I will get this cleared while working on some other optimization on this. Opening a tracker issue for this as well.

Reply:
#33 tracks this.